src/keywords.c

```diff
@@ -36,126 +36,128 @@
-                               ws_STORAGE,        ws_DIVIDER,
-                               ws_CONDUIT,        ws_PUMP,
-                               ws_ORIFICE,        ws_WEIR,
-                               ws_OUTLET,         ws_XSECTION,
-                               ws_TRANSECT,       ws_LOSS,
-                               ws_CONTROL,        ws_POLLUTANT,
-                               ws_LANDUSE,        ws_BUILDUP,
-                               ws_WASHOFF,        ws_COVERAGE,
-                               ws_INFLOW,         ws_DWF,
-                               ws_PATTERN,        ws_RDII, 
-                               ws_UNITHYD,        ws_LOADING,
-                               ws_TREATMENT,      ws_CURVE,
-                               ws_TIMESERIES,     ws_REPORT,
-                               ws_COORDINATE,     ws_VERTICES,
-                               ws_POLYGON,        ws_LABEL,
-                               ws_SYMBOL,         ws_BACKDROP, 
-                               ws_TAG,            ws_PROFILE,
-                               ws_MAP,            ws_LID_CONTROL,
-                               ws_LID_USAGE,      ws_GWF,                      //(5.1.007)
-                               ws_ADJUST,         ws_EVENT,                    //(5.1.011)
-                               NULL};                       
-char* SnowmeltWords[]      = { w_PLOWABLE, w_IMPERV, w_PERV, w_REMOVAL, NULL};
-char* TempKeyWords[]       = { w_TIMESERIES, w_FILE, w_WINDSPEED, w_SNOWMELT,
-                               w_ADC, NULL};
-char* TransectKeyWords[]   = { w_NC, w_X1, w_GR, NULL};
-char* TreatTypeWords[]     = { w_REMOVAL, w_CONCEN, NULL};
-char* UHTypeWords[]        = { w_SHORT, w_MEDIUM, w_LONG, NULL};
-char* VolUnitsWords[]      = { w_MGAL, w_MLTRS };
-char* VolUnitsWords2[]     = { w_GAL, w_LTR };
-char* WashoffTypeWords[]   = { w_NONE, w_EXP, w_RC, w_EMC, NULL};
-char* WeirTypeWords[]      = { w_TRANSVERSE, w_SIDEFLOW, w_VNOTCH,
-                               w_TRAPEZOIDAL, w_ROADWAY, NULL};                //(5.1.010)
-char* XsectTypeWords[]     = { w_DUMMY,           w_CIRCULAR,
-                               w_FILLED_CIRCULAR, w_RECT_CLOSED,
-                               w_RECT_OPEN,       w_TRAPEZOIDAL,
-                               w_TRIANGULAR,      w_PARABOLIC,
-                               w_POWERFUNC,       w_RECT_TRIANG,
-                               w_RECT_ROUND,      w_MOD_BASKET,
-                               w_HORIZELLIPSE,    w_VERTELLIPSE,
-                               w_ARCH,            w_EGGSHAPED,
-                               w_HORSESHOE,       w_GOTHIC,
-                               w_CATENARY,        w_SEMIELLIPTICAL,
-                               w_BASKETHANDLE,    w_SEMICIRCULAR,
-                               w_IRREGULAR,       w_CUSTOM,
-                               w_FORCE_MAIN,      NULL};
+// clang-format off
```

Comment:
@bemcdonnell in some cases we might want to preserve style like in this specific case.

```
// clang-format off

// clang-format on
```

To enable disable sections inside code